app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+from __future__ import annotations
+from app.errors import VaccineError, NotWearingMaskError
+from app.cafe import Cafe
+
+
+def get_amount_without_mask(friends: list) -> int:
+    return len([friend for friend in friends if not friend["wearing_a_mask"]])
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+
+    for i, friend in enumerate(friends):
```

Comment:
What is happening here? This can be done much simpler, do you really need enumeration?